ssz-rs-derive/src/lib.rs

```diff
@@ -228,6 +229,7 @@ fn derive_deserialize_impl(data: &Data) -> TokenStream {
                         let (_, end) = span[1];
 
                         container.__ssz_rs_set_by_index(index, &encoding[start..end])?;
+                        // checked_sub is unnecessary, as offsets are increasing
```

Comment:
Small nit: 
```suggestion
                        // checked subtraction is unnecessary, as offsets are increasing; qed
```
I would apply the same style to the other changed lines.